run-build-functions.sh

```diff
@@ -458,7 +458,7 @@ install_dependencies() {
   if [ -f package.json ]
   then
     restore_cwd_cache node_modules "node modules"
-    if [ -f yarn.lock ]
+    if [ -f yarn.lock ] || [ ! -z $USE_YARN ]
```

Comment:
Could you check that the value of this variable is `true`?
Like this, this allows for more values (including `false`) to be used as well.

Reply:
Ok, I changed it to check only for `false` or `true`